### PR TITLE
In TaskBar, count clearable tasks as "done" and make label consistent with progress bar

### LINKED ISSUE #6177: Task Bar counts failed tasks as "done" in progress bar, but not in the"x of y tasks complete" label
### Observed behavior

reported by @jtamiace during the 0.13.0 beta1 bug bash:

> 2 of 7 tasks failed, and the summary statement said '5 of 7 tasks complete'.
> 
> image
> 
> image

### Expected behavior

* Would be more accurate to say '7 of 7 tasks complete'.
* The task bar should also reflect this ratio, and _not_ incorporate actual number of resources or disk size

### User-facing consequences

confusing accounting of failed processes

### Steps to reproduce

start a number of imports, and cancel a few of them

### Context

0.13.0 beta 3

### PR DESCRIPTION
**Pull request by jonboiser**
### Summary
1. Changes how tasks are counted in TaskBar component. Canceled, failed, or complete tasks are counted as "Done" (as in they aren't running anymore). Queued and running tasks are counted as "in-progress". This is reflected in the text label as "(# clearable task) of (# all tasks) done". In the progress bar the numerator is (# clearable tasks) + (Sum of percentages of in-progess tasks).

Example: 1 canceled task + 1 completed task + 2 in-progress tasks

Screen Shot 2019-12-18 at 2 06 52 PM

Screen Shot 2019-12-18 at 2 06 48 PM

### Reviewer guidance

1. Try to run a diversity of Tasks; see if the TaskBar is consistent

### References

Fixes #6177 

----

### Contributor Checklist

PR process:

- [x] PR has the correct target branch and milestone
- [x] PR has 'needs review' or 'work-in-progress' label
- [ ] If PR is ready for review, a reviewer has been added. (Don't use 'Assignees')
- [ ] If this is an important user-facing change, PR or related issue has a 'changelog' label
- [ ] If this includes an internal dependency change, a link to the diff is provided

Testing:

- [ ] Contributor has fully tested the PR manually
- [ ] If there are any front-end changes, before/after screenshots are included
- [ ] Critical user journeys are covered by Gherkin stories
- [ ] Critical and brittle code paths are covered by unit tests

### Reviewer Checklist

- Automated test coverage is satisfactory
- PR is fully functional
- PR has been tested for [accessibility regressions](http://kolibri-dev.readthedocs.io/en/develop/manual_testing.html#accessibility-a11y-testing)
- External dependency files were updated if necessary (`yarn` and `pip`)
- Documentation is updated
- Contributor is in AUTHORS.md

### CODE REVIEW

File: kolibri/plugins/coach/assets/src/views/plan/PlanQuizPreviewPage.vue [hunk 1]
```diff
@@ -4,19 +4,22 @@
     :immersivePage="true"
     immersivePageIcon="arrow_back"
     :immersivePageRoute="toolbarRoute"
+    :immersivePagePrimary="true"
```
Comment (nucleogenesis) [inline kolibri/plugins/coach/assets/src/views/plan/PlanQuizPreviewPage.vue:7]: Are these changes supposed to be included here? They don't seem to be related to the task management.

**Review by nucleogenesis — CHANGES_REQUESTED** [review]
Just want to make sure the stuff in PlanQuizPreviewPage is supposed to be in this PR.

The task count works as expected though so that part looks good to me.

File: kolibri/plugins/coach/assets/src/views/plan/PlanQuizPreviewPage.vue (hunk 1 above)
Comment (jonboiser) [reply kolibri/plugins/coach/assets/src/views/plan/PlanQuizPreviewPage.vue:7]: Yeah, I'm throwing these changes in to fix #6395. This makes the preview page of the Quiz-creation workflow match the preview page of the Lesson-creation workflow. It must have gone under the radar in testing.

Comment (nucleogenesis) [reply kolibri/plugins/coach/assets/src/views/plan/PlanQuizPreviewPage.vue:7]: LGTM!

**Review by nucleogenesis — APPROVED** [review]
All set - both look and work properly.

**Comment by rtibbles** [comment]
Merging to avoid RPi bottleneck.

File: kolibri/plugins/coach/assets/src/views/plan/PlanQuizPreviewPage.vue (hunk 1 above)
Comment (indirectlylit) [reply kolibri/plugins/coach/assets/src/views/plan/PlanQuizPreviewPage.vue:7]: Definitely appreciate the drive-by fixes.

meta note for the future:

Generally we should try to keep independent changes in independent PRs to make code reviews and git archaeology easier. For example, smaller PRs play more nicely with using `git bisect` to identify the cause of regressions

However, I totally understand that it's usually quicker to throw multiple changes into a single omnibus PR, and it's crunch time.

One strategy I sometimes use is to make a bunch of commits related to multiple fixes on a single local branch, and then squash/re-arrange/cherry-pick them them into two or three separate branches as independent PRs. I personally use https://gitup.co/ which makes this kind of history rewriting super easy...

**Pull request merged**